service/build.gradle

```diff
@@ -30,10 +30,19 @@ dependencies {
 	implementation 'ch.qos.logback.contrib:logback-json-classic:0.1.5'
 	implementation 'ch.qos.logback.contrib:logback-jackson:0.1.5'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
+	implementation 'org.apache.qpid:qpid-jms-client:0.51.0'
+	implementation 'org.apache.httpcomponents:httpclient:4.5.12'
```

Comment:
Are we using httpclient anywhere?

Reply:
In utils/jmsReader.java used CharEncoding replaced with standard charset